src/actions/auth.js

```diff
@@ -454,7 +454,7 @@ export const handleRedirectResult = (dispatch, firebase, authData) => {
 export const init = (dispatch, firebase) => {
   // exit if auth does not exist
   if (!firebase.auth) {
-    return
+    throw new Error('"firebase/auth" must be imported to enable authentication')
```

Comment:
Will this be called and throw even if the intention is to not use auth? Maybe we should throw only if `userProfile` is also set? That way the error throws if you are attempting to use a feature which requires auth, otherwise just skips the logic

Reply:
I just noticed that `initializeAuth` props in ReactReduxFirebaseProvider is set to `true` by default, and that also took me time to realize how I forgot to include `initializeAuth={true}` in  props and it still initialize auth for me. Which made me thought only user specified `initializeAuth={true}` will certainly have an intention to use auth feature.

Now, we have to assume `initializeAuth` will run by default. I think it could be a good idea to exhaustively check for auth related config instead.